src/spec/integration/export.spec.coffee

```diff
@@ -72,7 +72,7 @@ describe 'Export integration tests', ->
   , 60000 # 60sec
 
 
-  it 'should inform about a bad header in the template', (done) ->
+  xit 'should inform about a bad header in the template', (done) ->
```

Comment:
Ahh, pardon .. I disabled them because of testing and then I forgot to enable them again, now it should be ok